README.md

```diff
@@ -164,12 +164,8 @@ This repo contains packages around SwapKit sdk and its integrations with differe
 ### Installation
 
 ```bash
-yarn install; yarn build
+pnpm install; pnpm build
```

Comment:
What do you think abour a ##Prerequisite section informing that you should have pnpm installed?